cmake/Format.cmake

```
@@ -55,10 +68,37 @@ set(
     ${PROJECT_SOURCE_DIR}/example/[^\.]*.cpp ${PROJECT_SOURCE_DIR}/example/[^\.]*.hpp
 )
 
+# Specifically trying to exclude external here - I'm not sure if there's a better way
+set(
+    PY_GLOBS
+    ${PROJECT_SOURCE_DIR}/scripts/[^\.]*.py
+    ${PROJECT_SOURCE_DIR}/tst/[^\.]*.py
+    ${PROJECT_SOURCE_DIR}/example/[^\.]*.py
+)
+
 if (CMAKE_VERSION VERSION_LESS "3.12.0")
     file(GLOB_RECURSE FORMAT_SOURCES ${GLOBS})
+    file(GLOR_RECURSE PY_FORMAT_SOURCES ${PY_GLOBS})
 else()
     file(GLOB_RECURSE FORMAT_SOURCES CONFIGURE_DEPENDS ${GLOBS})
+    file(GLOB_RECURSE PY_FORMAT_SOURCES CONFIGURE_DEPENDS ${PY_GLOBS})
 endif()
```

```suggestion
file(GLOB_RECURSE FORMAT_SOURCES CONFIGURE_DEPENDS ${GLOBS})
file(GLOB_RECURSE PY_FORMAT_SOURCES CONFIGURE_DEPENDS ${PY_GLOBS})
```

Given the latest Kokkos submodule update the min. required CMake version is now > 3.12 so we don't need this conditional any more.